### DISCUSSION TITLE
Pixijs alpha and animations in containers

### DISCUSSION THREAD

**Discussion by filipe-costa** [pixijs/pixijs#8025] [Q&A] [answered]
Hello all,

I am trying to create a popup, and I have done the necessary functionality. But, unfortunately, I am facing a minor issue where the alpha animation applied to the topmost container affects the children components.

Is there any way to stop this from propagating down the children elements or any other workaround using a different object?

I appreciate any help you can provide.

**Comment by bigtimebuddy** [comment]
Maybe this will help clarify:
https://jsfiddle.net/bigtimebuddy/v312Lp4n/

**Reply by filipe-costa** [reply]
> 

Thanks a lot for this; how would one manipulate the filter to create some fadeIn animation?

I have created a gif from what I am working on, and I have uploaded it to [Gyazo](https://gyazo.com/a3583bca3e40384fae4e7c20b1ae60c7).

The gif describes my problem, but I will also create a minimum reproducible demo for what I am trying to achieve later today.

**Comment by bigtimebuddy** [comment, marked as answer]
Instead of animating the `alpha` property on the Container, animate the `alpha` property on the filter:

```js
const filter = new PIXI.filters.AlphaFilter(1);

container.filters = [filter];
filter.alpha = 0.5; // <-- tween this property
```

**Reply by filipe-costa** [reply]
@bigtimebuddy , thanks a lot for this Matt! It worked perfectly fine.
